docs/axelar/bridge-tokens-axelar.md

```diff
@@ -0,0 +1,172 @@
+- Funded wallets on both chains.
+  - **ETH Sepolia Faucet:** [alchemy.com/faucets/ethereum-sepolia](https://www.alchemy.com/faucets/ethereum-sepolia)
+  - **XRPL Testnet Faucet:** [faucet.tequ.dev](https://faucet.tequ.dev/)
+- [XRPL JS Library](https://js.xrpl.org/)
+
+
+## Steps
+
+### Bridge ETH from Ethereum to XRPL
+
+1. Create a trust line on XRPL `testnet` between the Axelar multisig account and the destination account, using the `TrustSet` transaction.
+
+```javascript
+{
+    "TransactionType": "TrustSet",
+    "Account": user.address, // the XRPL address of the ETH transfer recipient
+    "LimitAmount": {
+        "currency": "ETH",
+        "issuer": "rfEf91bLxrTVC76vw1W3Ur8Jk4Lwujskmb",
+        "value": "10000000000",
+    },
+    ...
+},
+```
+
+2. Wrap ETH on Ethereum `Sepolia`. You can do this using the Foundry `cast` command, or directly on Etherscan.
+
+    - Using Foundry's `cast` command:
+  
+    ```sh
+    WETH=0x7b79995e5f793a07bc00c21412e50ecae098e7f9
+    cast send $WETH "deposit()" --value 0.1ether --private-key $PRIVATE_KEY --rpc-url $SEPOLIA_RPC_URL
+    ```
+    
+    - Using [Etherescan](https://sepolia.etherscan.io/token/0x7b79995e5f793a07bc00c21412e50ecae098e7f9#writeContract), select **Connect to Web3** to connect your wallet. Expand the **deposit** dropdown and type the amount of ETH you want to bridge. Click **Write** to perform the transaction.
+
+3. Approve the `AxelarGateway` contract to spend your WETH on Ethereum `Sepolia`:
+    
+    - Using Foundry's `cast` command:
+
+    ```sh
+    WETH=0x7b79995e5f793a07bc00c21412e50ecae098e7f9
+    AXELAR_GATEWAY=0xAABdd46ba1B3147d0Cf6aCc9605a74fE8668fC74
+    cast send $WETH "approve(address guy, uint256 amount)" $AXELAR_GATEWAY $(cast to-wei 0.1) --private-key $PRIVATE_KEY --rpc-url $SEPOLIA_RPC_URL
+    ```
+
+    - Using [Etherescan](https://sepolia.etherscan.io/token/0x7b79995e5f793a07bc00c21412e50ecae098e7f9#writeContract), select **Connect to Web3** to connect your wallet. Expand the **approve** dropdown, enter `AxelarGateway`'s address (`0x832B2dfe8a225550C144626111660f61B8690efD`), and the amount of ETH to bridge under **deposit**. Click **Write** to perform the transcation.
+
+4. On Ethereum Sepolia, call `AxelarGateway.sendToken()` to bridge your WETH.
+
+    ```sh
+    AXELAR_GATEWAY=0xAABdd46ba1B3147d0Cf6aCc9605a74fE8668fC74
+    XRPL_DESTINATION= # your XRPL recipient address
+    cast send $AXELAR_GATEWAY "sendToken(string destinationChain, string destinationAddress, string symbol, uint256 amount)" "xrpl" $XRPL_DESTINATION "WETH" $(cast to-wei 0.1) --private-key $PRIVATE_KEY --rpc-url $SEPOLIA_RPC_URL
+    ```
+
+5. The destination address should receive the bridged ETH in a few minutes. You can check for a successful transaction on the [XRPL Testnet explorer](https://testnet.xrpl.org/).
+
+
+### Bridge ETH from XRPL to Ethereum
```

Comment:
Change to "Bridge wrapped ETH back to Ethereum Sepolia"
Note that there is no native ETH on the XRPL. In this section, we are just bridging back the wrapped ETH.